src/pvnet_app/data/satellite.py

```diff
@@ -232,6 +239,66 @@ def check_model_satellite_inputs_available(
     return available
 
 
+def get_pvnet_spatial_slice(
```

Comment:
could you add something about satellite in here. Like `get_satellite_spatial_slice_for_gsp_locations`